### ISSUE TITLE
Where can I get Indonesian lexicon?

### ISSUE THREAD

**Issue by rivanfebrian123** [metalcorebear/NRCLex#17]
Hi! I appreciate your work, but I been struggling on finding the Indonesian lexicon like the one provided in the original website.

I want to use this library for my research. Thank you

**Comment by metalcorebear** [comment]
Good morning. I'm glad to hear that you find the library useful. Unfortunately, I'm not aware of any Indonesian lexicon that would be compatible with this library.

**Comment by rivanfebrian123** [comment]
That's fine! I found the way to use the original lexicon in your library.

Actually, when I open this issue, I was in a class and thinking that the deadline would coming anytime soon, but it's not. I asked my friends like "should I open this issue" they were like "sure go ahead".

Last night, I explored your code and found the lexicon file, and I was like "wait, is that even legal?". So I emailed him

![图片](https://user-images.githubusercontent.com/6300910/206309006-b99ec2dc-812b-4918-89e8-ebdb7368d4b4.png)

Yeah, the terms of use is too strict.
